### PR TITLE
add libtiff/4.0.9, libtiff/4.0.8

### PR DESCRIPTION
**Pull request by theirix**
Specify library name and version:  **libtiff/4.0.9**, **libtiff/4.0.8**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/libtiff/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    author = "Bincrafters <bincrafters@gmail.com>"
+    license = "MIT"
+    homepage = "http://www.simplesystems.org/libtiff"
+    exports = ["LICENSE.md"]
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {'shared': False, 'fPIC': True}
+    requires = "zlib/1.2.11@conan/stable"
```
Comment (uilianries) [inline recipes/libtiff/all/conanfile.py]:
```suggestion
    requires = "zlib/1.2.11"
```

File: recipes/libtiff/all/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+from conans import ConanFile, CMake, tools
+import os
+import shutil
+
+
+class LibtiffConan(ConanFile):
+    name = "libtiff"
+    description = "Library for Tag Image File Format (TIFF)"
+    url = "http://github.com/bincrafters/conan-tiff"
```
Comment (uilianries) [inline recipes/libtiff/all/conanfile.py]:
```suggestion
    url = "https://github.com/conan-io/conan-center-index"
```

File: recipes/libtiff/all/conanfile.py [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    author = "Bincrafters <bincrafters@gmail.com>"
+    license = "MIT"
+    homepage = "http://www.simplesystems.org/libtiff"
+    exports = ["LICENSE.md"]
```
Comment (uilianries) [inline recipes/libtiff/all/conanfile.py]:
```suggestion
```
No longer required

File: recipes/libtiff/all/conanfile.py (hunk 1 above)
Comment (theirix) [reply recipes/libtiff/all/conanfile.py]: Committed at the same time, thanks!

**Comment by uilianries** [comment]
Let's restart this one. Jenkins is jammed.

**Comment by lasote** [comment]
@theirix it is a "bug" that only is showing in the count the binaries built in this "reload" of the build script. It is caching the binaries for the same commit. We are going to open an issue because it is confusing, but the build is ok, all the packages are there.

**Comment by uilianries** [comment]
Related issue: https://github.com/conan-io/conan-center-index/issues/114

File: recipes/libtiff/all/conanfile.py [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+import shutil
+
+
+class LibtiffConan(ConanFile):
+    name = "libtiff"
+    description = "Library for Tag Image File Format (TIFF)"
+    url = "https://github.com/conan-io/conan-center-index"
+    author = "Bincrafters <bincrafters@gmail.com>"
+    license = "MIT"
+    homepage = "http://www.simplesystems.org/libtiff"
```
Comment (uilianries) [inline recipes/libtiff/all/conanfile.py]:
```suggestion
    homepage = "http://www.simplesystems.org/libtiff"
    topics = ("tiff", "image", "bigtiff", "tagged-image-file-format")
```

Comment (uilianries) [reply recipes/libtiff/all/conanfile.py]: add topics

**Comment by uilianries** [comment]
Let's re-start this one ...

**Pull request merged**